lib/plugins/aws/package/compile/events/kafka.js

```diff
@@ -0,0 +1,212 @@
+'use strict';
+class AwsCompileKafkaEvents {
```

Comment:
Let's add a line space between `'use strict'` and code (sorry we should have a lint rule for that)